changes/541.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+- Created a new exception class InvalidSupportPackageURL
+- MissingNetworkResourceError is caught in install_app_support_package and InvalidSupportPackageURL is raised
+- Test Case test_support_package_url_with_invalid_url added to check the new exception
```

Comment:
The purpose of this file is to provide the text for the release note associated with this feature. You shouldn't be describing the implementation approach, or the code changes - you should be describing the feature that has been added or bug that has been resolved, as experienced by the end user.

Reply:
Got it, I'll revise this